README.md

```diff
@@ -67,17 +67,17 @@ You can access the data by using this link:
 
 - Authentication
 - Periodical Table - History
-- Subcription based data
-- payment system (stripe or paytm)
+- Subscription-based data
+- payment system (stripe or Paytm)
 - Newsletter
 - Contact us page
 
 ## Expected JSON Data Format
 
-```json
+```JSON
 {
   "name": "Hydrogen",
-  "appearance": "colorless gas",
+  "Appearance": "colorless gas",
```

Comment:
This is the data coming from the API so this remains the same. Correct this part then I will merge your PR.

Reply:
I made relevant changes to the readme.md file. please see